### PR TITLE
Fast GELU CUDA kernel

### PR DESCRIPTION
**Pull request by AndreSlavescu**
Changes:
- GELU with tanh as approximation
- using faster approximated tanh for improved speed

Ideas: 
- give user option to run faster gelu for less accuracy? Currently torch.nn.GELU() is being dispatched and all other variations in the activation registry are unused

Todos:
- run tests on different input sizes and benchmark performance trade-off

### CODE REVIEW

**Comment by WoosukKwon** [comment]
Closed as this has been implemented by #828
